ci/github/get_hdf5.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+# vendored from https://github.com/h5py/h5py/blob/master/ci/get_hdf5_if_needed.sh
```

Comment:
This comment does't seem to be true anymore (it seems this file experienced quite some changes).

That's not per se bad - but maybe we can change the wording to "inspired by ..." ?

Reply:
Fixed